voltixApp/VoltixApp/Services/BalanceService.swift

```diff
@@ -58,9 +66,9 @@ class BalanceService {
         case .gaiaChain:
             let atomBalance =  try await gaia.fetchBalances(address: coin.address)
             var balanceFiat: String = .empty
-            balanceFiat = atomBalance.atomBalanceInFiat(price: coin.priceRate) ?? "$ 0,00"
-            coin.rawBalance = atomBalance.atomBalance() ?? "0.0"
-            let coinBalance = atomBalance.formattedAtomBalance() ?? "0.0"
+            balanceFiat = atomBalance.coinBalanceInFiat(price: coin.priceRate,coin: coin) ?? "$ 0,00"
```

Comment:
I am replacing all of these because we don't need to use coinBalance In Fiat.

```
//
//  THORBalanceExtension.swift
//  VoltixApp
//
//  Created by Johnny Luo on 22/3/2024.
//

import Foundation

extension [CosmosBalance] {
    func balance(denom: String) -> String {
        for balance in self {
            if balance.denom.lowercased() == denom {
                return balance.amount
            }
        }
        return .zero
    }
}

```